### ISSUE TITLE
get element by id

### ISSUE THREAD

**Issue by flip111** [elm-community/elm-list-extra#16]
Hey i made this small function to get an element by id, perhaps it can be added?

``` elm
(!!) : List a -> Int -> Maybe a
(!!) list index =
  Array.get index (Array.fromList list)

infixr 9 !!
```

I liked the `!!` syntax as used by haskell. Only thing i'm not sure of if i set the right priority with the `infixr`

**Comment by rehno-lindeque** [comment]
Thanks, please submit a PR and remember to export the function, add a `@doc` + `elm build` to test it works. I think that probably the most efficient implementation will be something like:

``` elm
List.head <| List.drop index list
```

For the infix priority, I left it unspecified (default is `infixl 9` afaik) when I implemented [(?)](https://github.com/circuithub/elm-maybe-extra/blob/1.5.0/src/Maybe/Extra.elm#L31) on `Maybe`. I suspect that's also a good default for `!!` since you most likely what `!!` to bracket on the left for multidimensional arrays. E.g.

``` elm
matrix !! 3 !! 2 -- row 3, column 2 in a row-major 2x2 matrix
```

**Comment by flip111** [comment]
what do you mean with `export` and `elm build` ?

**Comment by rehno-lindeque** [comment]
Just basically add the function in the export list you'll see at the top of the file:

``` elm
module List.Extra
  ( last
  , ... 
  ,  <- Please add it here
  )  where
```

and then also build the package to make sure it compiles (We get many PR's that don't compile or don't export the newly added function :)).

**Comment by rtfeldman** [comment]
Here's a complete implementation, if anyone would like to make a PR:

``` elm
{-| Returns Just the element at the given index in the list,
or Nothing if the list is empty.
-}
getAt : Int -> List a -> Maybe a
getAt index list =
    List.head (List.drop (index) list)
```

**Comment by twopoint718** [comment]
I merged #22

**Comment by rehno-lindeque** [comment]
Looks good, thank you for merging
